views/confirmation.ejs

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+  <div class="row justify-content-center py-5 text-center">
+    <div class="col-11 col-md-9" style="padding: 5rem 1rem 2rem; background-color: #EEF8F3; border-radius: 15px;">
+      <!-- Checkmark Icon -->
+      <div class="py-5">
+        <a href="/" style="color: #639C70;">
```

Comment:
do we want to come back to homepage when click on checkmark? @DariaBezuhla @lyrated

Reply:
I linked it on the error page, thought it would be better since the button is so far down and they maybe don't notice it :P for the confirmation we can just remove the link or link it to "Add new app"

Reply:
It would be nice if it would take the user to an overview of the added apps/some app list. 
I would rather remove it than link it to "Add new app" since there is already a button for that.